dataikuapi/dss/analysis.py

```diff
@@ -188,8 +188,9 @@ def create_prediction_ml_task(self,
         return mltask
 
     def create_clustering_ml_task(self,
-                                   ml_backend_type = "PY_MEMORY",
-                                   guess_policy = "KMEANS"):
+                                  ml_backend_type="PY_MEMORY",
+                                  guess_policy="KMEANS",
+                                  wait_guess_complete=True):
```

Comment:
IIUC, this is
- aligned with prediction
- not what was done in previous versions

Could say that on the PR desc

Reply:
done